scripts/skills/actives/rf_pocket_sand_skill.nut

```diff
@@ -0,0 +1,87 @@
+this.rf_pocket_sand_skill <- this.inherit("scripts/skills/actives/throw_dirt_skill", {
+	m = {
+		RemainingUses = 0
+	},
+
```

Comment:
Vanilla doesn't put an empty space between the `m` table and `function create` so let's keep it like that.